recipes/pynvjitlink/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "pynvjitlink" %}
+{% set version = "0.1.0.dev1" %}
+
+package:
+  name: pynvjitlink
+  version: {{ version }}
+
+source:
+  git_url: git@github.com:rapidsai/nvjitlink.git
+  git_rev: 2aca1ce3c56aa90b6f7c93d43085e25f50670838
+  git_depth: 1
+
+
+build:
+  script:
+    # This is a workaround for library search path issues with the current
+    # cuda-nvcc package. They should be fixed by
+    # https://github.com/conda-forge/cuda-nvcc-feedstock/pull/24, at which
+    # point we can remove this.
+    - export CMAKE_PREFIX_PATH=${PREFIX}/targets/x86_64-linux/lib:$CMAKE_PREFIX_PATH
+    - {{ PYTHON }} -m pip install . -vv
```

Since PR ( https://github.com/conda-forge/cuda-nvcc-feedstock/pull/24 ) landed, this can be dropped

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```